src/manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<updates>
+    <!-- update sample from Joomla at https://docs.joomla.org/Deploying_an_Update_Server
+    <update>
+        <name>Weblinks Extension Package</name>
+        <description>Joomla! CMS Weblinks Package</description>
+        <element>pkg_weblinks</element>
+        <type>package</type>
+        <version>3.6.0</version>
+        <client>site</client>
+        <infourl title="Weblinks Extension Package">https://github.com/joomla-extensions/weblinks/releases/tag/3.6.0</infourl>
+        <downloads>
+            <downloadurl type="full" format="zip">https://downloads.joomla.org/extensions/weblinks/3-6-0/pkg-weblinks-3.6.0.zip</downloadurl>
```

Comment:
I am not sure what a link this is, but I believe this should refer to the zip file of the plugin to be downloaded. In this case it could be `https://www.newsletter2go.de/plugins/deploy_new/Joomla/Joomla_latest.zip`

Reply:
Ok, i'll point to this place for the updated *.zip